root.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  backend_state_lock   = "tdr-bootstrap-terraform-state-lock"
```

Comment:
The state bucket and state lock will not be the backend. It this repo will have its own state lock and state bucket.

Reply:
Yeah, good point. I have replaced it with our own state and state-lock